spec/support/vendor_direct_fulfillment_orders/v20211228_get_order.json

```diff
@@ -0,0 +1,107 @@
+{
+  "purchaseOrderNumber": "2JK3S9VC",
+  "orderDetails": {
+    "customerOrderNumber": "123-ABC",
+    "orderDate": "2020-02-20T13:51:00Z",
+    "shipmentDetails": {
+      "isPriorityShipment": false,
+      "isScheduledDeliveryShipment": false,
+      "isPslipRequired": true,
+      "isGift": false,
+      "shipMethod": "UPS_2ND",
+      "shipmentDates": {
+        "requiredShipDate": "2020-02-21T00:00:00Z",
+        "promisedDeliveryDate": "2020-02-24T00:00:00Z"
+      },
+      "messageToCustomer": "This shipment completes your order. You can always check the status of your orders from the \"Your Account\" link at the top of each page of our site.Thank you for shopping at Amazon.com"
+    },
+    "taxTotal": {
+      "taxLineItem": [
+        {
+          "taxRate": "0.1",
+          "taxAmount": {
+            "currencyCode": "USD",
+            "amount": "190"
+          },
+          "type": "TOTAL"
+        }
+      ]
+    },
+    "sellingParty": {
+      "partyId": "999US"
+    },
+    "shipFromParty": {
+      "partyId": "ABCD"
+    },
+    "shipToParty": {
+      "name": "John Doe",
```

Comment:
hope this is a dummy data, if not need to change the shipping details here

Reply:
Yes its dummy data !